src/main/java/innovimax/mixthem/operation/AltOperation.java

```diff
@@ -0,0 +1,23 @@
+package innovimax.mixthem.operation;
+
+import innovimax.mixthem.arguments.ParamValue;
+
+public enum AltOperation { 
+  DEFAULT_RANDOM_SEED(1789);
+
+  private ParamValue value;
```

Comment:
why paramValue? why not just int ?

Reply:
Used as default value for parameters in Map.getOrDefault().
Another way is to create default value when used with ParamValue.createInt().